src/loader.js

```diff
@@ -88,22 +88,26 @@
         return entry.module;
       });
 
+    // @ifdef STRICT
```

Comment:
I guess let's stick with this method for now then actually and we can easily re-evaluate in future.